src/main/java/core/basesyntax/operation/OperationStrategyImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
```

Comment:
strategy class should only return the handler. Add separate ShopService for processing transactions